gatsby-node.js

```diff
@@ -26,20 +65,10 @@ const createCareerPages = async ({ actions }) => {
   const jobsParse = xmlParser.parse(jobsXml);
   const positions = jobsParse['workzag-jobs'].position;
 
-  const slugifyPositionName = (name) => {
```

Comment:
I installed `slugify`. Actually it should be installed already (implicite) because it's a dependency of the remark plugin. I think it makes sense to swap our custom code withe the "standard" library of creating slugs.